Controller/RedirectRouteImportController.php

```diff
@@ -70,12 +70,19 @@ public function getLocale(Request $request)
      */
     public function postAction(Request $request)
     {
-        if (!$request->files->has('redirectRoutes')) {
+        if (!$request->files->has('files')) {
             return new JsonResponse(null, Response::HTTP_BAD_REQUEST);
         }
 
-        /** @var UploadedFile $uploadedFile */
-        $uploadedFile = $request->files->get('redirectRoutes');
+        /** @var UploadedFile[] $uploadedFiles */
+        $uploadedFiles = $request->files->get('files');
+
+        if (\count($uploadedFiles) < 1) {
```

Comment:
The toolbarAction should have an option to not send it as an array, or I think it should be the default and an option `multiple` should be used.